Cargo.toml

```diff
@@ -1,14 +1,19 @@
 [package]
 name = "blockstack-core"
 version = "0.0.1"
-authors = ["Jude Nelson <jude@blockstack.com>", "Aaron Blankstein <aaron@blockstack.com>"]
+authors = [ "Jude Nelson <jude@blockstack.com>",
+            "Aaron Blankstein <aaron@blockstack.com>",
+            "Ludo Galabru <ludovic@blockstack.com>" ]
 license = "GPLv3"
 homepage = "https://github.com/blockstack/blockstack-core"
 repository = "https://github.com/blockstack/blockstack-core"
 description = "Reference implementation of Blockstack Core"
 keywords = [ "bitcoin", "crypto", "blockstack", "decentralized", "dapps", "blockchain" ]
 readme = "README.md"
 
+[profile.release]
+debug = true
```

Comment:
I think we should also add this config to the `testnet/src/Cargo.toml` (where the fields authors/version should be stripped)

Reply:
Seconded

Reply:
The top-level Cargo.toml overrides any profile settings in the other workspace Cargo.toml files -- so if we add that to the other Cargo.tomls, all we'll get is warnings about how they are ignored.